config/consensus.go

```diff
@@ -103,6 +103,10 @@ type ConsensusParams struct {
 	// a way of making the spender subsidize the cost of storing this transaction.
 	MinTxnFee uint64
 
+	// EnableFeePooling specifies that the sum of the fees in a
+	// group must exceed one MinTxnFee, rather check on each Txn.
```

Comment:
I think this comment is out-of-date with what you are doing: perhaps it should say "one MinTxnFee per transaction in group"?